docs/architecture/adr-027-deterministic-protobuf-serialization.md

```diff
@@ -11,7 +12,7 @@ Proposed
 ## Context
 
 [Protobuf](https://developers.google.com/protocol-buffers/docs/proto3)
-seralization is not unique (i.e. there exist a practically unlimited number of
+serialization is not unique (i.e. there exist a practically unlimited number of
 valid binary representations for a protobuf document)<sup>1</sup>. For signature
 verification in Cosmos SDK, signer and verifier need to agree on the same
 serialization of a SignDoc as defined in
```

Comment:
So, `SignDoc` will do serialization, correct?

Reply:
That's outside the scope of this PR, which only aims to clarify the rules for serialization.